### ISSUE TITLE
Staging Review finding: Screenshots must have descriptive alternative text.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#95787]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** Help users to... Know when their information is prefilled
**Staging Review ticket:** [AEDP, Design Patterns, Help users to... Know when their information is prefilled](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90181)

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**This is an issue with the:** 
**High-priority:** yes
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

You must provide alternative text that describes the content of an image, so that VFS team members who can't perceive an image visually will still be able to understand the information in the image. Currently, alt text is empty, just a restatement of the image caption, or not descriptive of the content displayed.

## Recommended action
Good alt text for a screenshot is a bit of an art. It's different than what's in the image caption, and more of a literal description of what's in the image itself. But you don't need to describe everything in the image, just the parts that are contextually relevant.

For example, for the [personal information page screenshot](https://design.va.gov/patterns/help-users-to/know-when-their-information-is-prefilled#personal-information-page), I might write alt text like:

```
Veteran information page in a form flow with an uneditable card containing the Veteran's name, privacy masked Social Security number, date of birth, and gender. After the card is a note explaining why the content can't be edited and how to update this information.
```

Each screenshot must have text that's comparably descriptive included in the `alt` attribute of the `img` element.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.1.1 Non-text Content - Level A](https://www.w3.org/TR/WCAG21/#non-text-content)
- **Modality:** Screen reader
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
